python/packages/autogen-core/docs/src/user-guide/core-user-guide/design-patterns/multiple-agents-concurrently.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Multiple Agents Concurrently"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 118,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import asyncio\n",
+    "from dataclasses import dataclass\n",
+    "\n",
+    "from autogen_core.application import SingleThreadedAgentRuntime\n",
+    "from autogen_core.base import AgentId, MessageContext, TopicId\n",
+    "from autogen_core.components import (\n",
+    "    DefaultTopicId,\n",
+    "    RoutedAgent,\n",
+    "    default_subscription,\n",
+    "    message_handler,\n",
+    "    type_subscription,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "@dataclass\n",
+    "class Task:\n",
+    "    task_id: str\n",
+    "\n",
+    "\n",
+    "@dataclass\n",
+    "class TaskResponse:\n",
+    "    task_id: str\n",
+    "    result: str"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Single Message & Multiple Processors\n",
+    "The first pattern shows how a single message can be processed by multiple agents simultaneously:\n",
+    "\n",
+    "- Each `Processor` agent subscribes to the default topic using the {py:meth}`~autogen_core.components.default_subscription` decorator\n",
```

Comment:
Add to this section a note stating there is an alternative way to add subscription for an agent type without using the class decorator, so an agent class can be used for different subscriptions. Point to the guide section: https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/framework/message-and-communication.html#subscribe-and-publish-to-topics

Reply:
use 

````

```{note}
Add note here
```

```

To highlight this